### ISSUE TITLE
I met error when doing 「6.Run these commands to connect to Exchange Online」

### ISSUE THREAD

**Issue by syuu2017** [MicrosoftDocs/OfficeDocs-Enterprise#237]
If I do 1→6、there will be success.
But if I do 1→2→3→4→5→6、then i will meet error as contents below.

PS C:\Users\Administrator&gt; $exchangeSession = New-PSSession -ConfigurationName Microsoft.Exchange -ConnectionUri "https://outlook.office365.com/powershell-liveid/" -Credential $credential -Authentication "Basic
" -AllowRedirection
New-PSSession : [outlook.office365.com] リモート サーバー outlook.office365.com への接続に失敗し、次のエラー メッセージが返されました: アクセスが拒否されました。詳細については、about_Remote_Troubleshooting の
ヘルプ トピックを参照してください。
発生場所 行:1 文字:20
+ ... geSession = New-PSSession -ConfigurationName Microsoft.Exchange -Conn ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OpenError: (System.Manageme....RemoteRunspace:RemoteRunspace) [New-PSSession], PSRemotingTransportException
    + FullyQualifiedErrorId : AccessDenied,PSSessionOpenFailed

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: 0c551eb1-7ac5-cb02-cd1b-d67ca3886d85
* Version Independent ID: fb690e2e-420e-f697-60f2-39520283d385
* Content: [単一の Windows PowerShell ウィンドウですべての Office 365 サービスに接続する](https://docs.microsoft.com/ja-jp/office365/enterprise/powershell/connect-to-all-office-365-services-in-a-single-windows-powershell-window)
* Content Source: [Enterprise/powershell/connect-to-all-office-365-services-in-a-single-windows-powershell-window.md](https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/blob/live/Enterprise/powershell/connect-to-all-office-365-services-in-a-single-windows-powershell-window.md)
* Service: **o365-administration**
* GitHub Login: @JoeDavies-MSFT
* Microsoft Alias: **josephd**

**Comment by syuu2017** [comment]
sorry, I didn't write the contents well.

**Comment by robmazz** [comment]
@syuu2017 - thanks for the feedback, I'll get this over to the Enterprise content team for clarification.

**Comment by JoeDavies-MSFT** [comment]
@syuu2017 I have tried both methods (1-2-3-4-5-6) and 1-6 and both worked for me without the error above. I have added the "-DisableErrorChecking" parameter for the Exchange "Import-Session" command. Try this. See https://docs.microsoft.com/powershell/exchange/exchange-online/connect-to-exchange-online-powershell/connect-to-exchange-online-powershell?view=exchange-ps#how-do-you-know-this-worked for additional troubleshooting steps. If all else fails, use https://support.office.com/home/contact to create a support ticket.
